PROJECT_PROGRESSION.md

```diff
@@ -13,34 +13,16 @@ This proposal has been modified from the [CNCF process documentation](https://gi
 This governance policy sets forth the proposal process for projects to be accepted into the Foundation. The process is the same for both existing projects which seek to move into the Foundation, and new projects to be formed within the Foundation.
 
 ### Project Proposal Requirements
-Projects must be proposed via GitHub. Project proposals submitted to the Foundation must provide the following information to the best of their ability:
-
-* name of project
-* project description (what it does, why it is valuable, origin and history)
-* statement on alignment with Foundation charter mission
-* link to *current* Code of Conduct
-* sponsor from CPC, if identified (a sponsor helps mentor projects)
-* project license 
-* source control (GitHub by default)
-* issue tracker (GitHub by default)
-* external dependencies (including licenses)
-* release methodology and mechanics
-* names of initial committers, if different from those submitting proposal
-* briefly describe the project's leadership team and decision-making process
-* preferred maturity level (see stages below)
-* list of project's official communication channels (slack, irc, mailing lists)
-* link to project's website 
-* links to social media accounts
-* existing financial sponsorship
-* infrastructure needs or requests 
-
-### Project Acceptance Process
-* Projects are required to present their proposal at a CPC meeting
-* The CPC may ask for changes to bring the project into better alignment with the Foundation (adding a governance document to a repository or adopting a more stringent Code of Conduct, for example).
- * The project will need to make these changes in order to progress further.
-* Projects get accepted via a 2/3 supermajority vote of the CPC.
-* The proposal document will be finalized as a project charter. This charter document must be included in the project's main repository.
-* The CPC will determine the appropriate initial stage for the project. The project can apply for a different stage via the review process. 
+
+New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.
```

Comment:
This paragraph seems to be redundant, as steps are described in Application Phases.
How about we remove it partly/completely, and add info in phases instead?

Reply:
Some of the copy about incubation process isn't documented anywhere else. Perhaps that should be a separate section